cmd/dapr.go

```diff
@@ -41,13 +41,26 @@ var RootCmd = &cobra.Command{
 Distributed Application Runtime`,
 }
 
-var logAsJSON bool
+type daprVersion struct {
+	CLI     string `json:"Cli version"`
```

Comment:
nit: json tags should have json naming convention. `cli_version`, `runtime_version`, etc.

Reply:
would prefer `cliVersion` and `runtimeVersion`

Reply:
classical snake_case vs camelCase issue, `dapr` seems to on the sides of camelCase.

Reply:
Anything works as long as we are consistent 😄

Reply:
@fanfanbj can you please update the json string to what @mukundansundar suggested?